### ISSUE TITLE
Store tries to read JSON file with wrong filename

### ISSUE THREAD

**Issue by bampakoa** [sindresorhus/electron-store#30]
Hi, 

I recently upgraded my application to version **1.3.0** of the library and the following initialization fails:

```
const store = new Store({
    name: 'maps',
    cwd: `${__dirname}/assets`
});
```

with error:

`Error: ENOENT, assets\maps.json.714842314 not found in D:\myapp\resources\app.asar`

The file is **maps.json** and exists inside **assets** folder. When I build my application using `electron-packager`, for some reason, `electron-store` tries to read the file as `maps.json.<random number>` .

Maybe because of atomic writing feature in the `conf` module? If I lock `conf` in version 1.1.2 it works fine.

Thanks

**Comment by sindresorhus** [comment]
Yeah, looks like it's because of the atomic writing. Sounds like a bug in [asar](https://github.com/electron/asar). Try opening an issue there instead.

**Comment by bampakoa** [comment]
@sindresorhus Let me clarify that when the app is packaged, the filename is still the same inside the asar file (maps.json). Do you think that I should still file an issue in asar repo?

Many thanks

**Comment by sindresorhus** [comment]
It works without Asar, so yes, it probably because Asar patches the `fs` methods.

**Comment by bampakoa** [comment]
I am closing this now since the problem seems to have been resolved in Electron 4.0
